content/sensu-enterprise/2.6/guides/using-contact-routing.md

```diff
@@ -0,0 +1,619 @@
+
+Before diving into this guide, we recommend having the following components ready:
+
+- A working Sensu Enterprise deployment
+- Two or more handlers configured
+
+If you've not already signed up for Sensu Enterprise, you can do so via [this link][1].
+
+In this guide we'll be using [Slack][2] and [Email][3] handlers for our demonstration.
+
+# Overview
+
+Every incident, outage or event has an ideal first responder. This can be either
+a team or individual with the knowledge to triage and address the issue. Sensu
+Enterprise contact routing makes it possible to assign checks to specific teams
+and/or individuals, reducing mean time to response and recovery (MTTR). Contact
+routing works with all of the Sensu Enterprise third-party notification and
+metric integrations.
+
+_NOTE: Sensu Enterprise supports contact routing for all integrations except
+EC2, Puppet, Chef, Flapjack, and Event Stream._
+
+In this guide we'll cover configuring and using Sensu Enterprise Contact Routing.
+
+## Contact Routing Basics
+
+In Sensu Enterprise Contact Routing, contacts are composed of a name and
+configuration overrides for one or more of Sensu Enterprise's built-in
+integrations or handler configurations. A contact in Sensu Enterprise is not too
+dissimilar from a contact on your phone, in which they have a name and one or
+more _identifiers_ for various communication **channels** (e.g. a **phone**
+_number_, **email** _address_, **Twitter** _username_, etc). As an example your contact may have the following **email** _address, **slack** _channel_, **pagerduty** _service\_key_, etc.
+
+### Contact Configuration
+
+Configuring a contact requires you to define the name of the contact and the
+services plus configuration override(s) for that contact.
+
+{{< highlight json >}}
+{
+  "contacts": {
+    "support": {
+      "email": {
+        "to": "support@sensuapp.com"
+      },
+      "slack": {
+        "channel": "#support"
+      }
+    }
+  }
+}
+{{< /highlight >}}
+
+In the example above we have a contact named **support**. For **support** we
+have an override configuration for **email** and **slack**. In our example we're having
+this contact email _support@sensuapp.com_ instead of the email integration's
+default **to** address. For Slack, we're defining the channel the event should
+post to, in this case _#support_.
+
+### Check Configuration
+
+Once a contact has been defined, we can now apply which contact(s) to use in a
+check configuration. The use case for having a contact defined on a check could 
+be that the check is owned by a particular team or group. For instance:
+
+{{< highlight json >}}
+{
+  "checks": {
+    "example_check": {
+      "command": "do_something.rb",
+      "handlers": [
+        "email"
+      ],
+      "contacts": [
+        "support"
+      ]
+    }
+  }
+}
+{{< /highlight >}}
+
+### Client Configuration
+
+A client configuration can include a contact to use. This is helpful for defining ownership of the client itself.
```

Comment:
I don't think its clear what "defining ownership of the client" means. This seems like right place to describe how contacts at the client level become the defaults for events that this client creates. Those defaults can be overridden by contacts specified in the check definition.